### ISSUE TITLE
Does the llm not support models on hugging face (open source)?

### ISSUE THREAD

**Issue by bank010** [mem0ai/mem0#1734]
### 🚀 The feature

Does the llm not support models on hugging face (open source)?

### Motivation, pitch

Does the llm not support models on hugging face (open source)?

**Comment by roshanr11** [comment]
tldr - yes. 

---

Based on the docs right now, looks like there's support for embedding models: https://docs.mem0.ai/components/embedders/models/huggingface#hugging-face. 

---

However, from twitter: https://x.com/taranjeetio/status/1824113002641326245 I saw that LiteLLM support was added last week. Combine this with LiteLLM's docs: https://docs.litellm.ai/docs/providers/huggingface. 

---

Basically, you can do this as follows:

![image](https://github.com/user-attachments/assets/1c7bf3a0-98f9-49dd-8f54-be9bf8622ab5)

but replace "model":___ with "huggingface/<your_model_name>" such as "huggingface/facebook/blenderbot-400M-distill".

**Comment by parshvadaftari** [comment]
Closing as fixed.
